src/index.js

```diff
@@ -0,0 +1,133 @@
+import './style.css';
+import { postComments, getComments } from './comment.js';
+
+const url = 'https://api.tvmaze.com/shows';
+const countUrl = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/Q4ShCASnnzP55bg7hv5u/likes';
+const elem = document.querySelector('.cards');
+let data;
+
+function render(data) {
+  document.querySelector('h1').innerText += ` (${data.length})`;
+  elem.innerHTML = '';
+  for (let i = 0; i < data.length; i += 1) {
+    elem.innerHTML += `
+<div class="cards-info" id='movie-${data[i].id}'>
+         <img src=${data[i].image.medium} alt="" class="img">
+            <p class=description>${data[i].name}</p>
+<div class='likes-count'> 
+      <i class="fas fa-heart fa-3x" ></i>
+      <span>Likes-${data[i].likes}</span>
+      </div>
+            <div class="button">
+              <button class="cmntBtn-button">Comments</button>
+              <button class='reBtn'>Reservations</button>
+            </div>
+          </div>
+        </div>`;
+  }
+  async function count(obj) {
+    await fetch(countUrl,
+      {
+        method: 'POST',
+        headers: { 'Content-Type': 'application/json' },
+        body: JSON.stringify(obj),
+      });
+  }
+
+  document.querySelectorAll('i').forEach((like) => {
+    like.addEventListener('click', () => {
+      const obj = { item_id: like.id };
+      const id = Number(like.parentNode.parentNode.id.split('-')[1]);
+      const movie = data.find((x) => x.id === id);
+      movie.likes += 1;
+      obj.likes = movie.likes;
+      like.parentNode.querySelector('span').innerText = `Likes-${obj.likes}`;
+      count(obj);
+    });
+  });
+
+  const clearInputsFields = () => {
+    const userName = document.querySelector('#username');
+    const userComment = document.querySelector('#comment');
+    userName.value = '';
+    userComment.value = '';
+  };
+
+  document.querySelectorAll('.cmntBtn-button').forEach((item) => {
+    item.addEventListener('click', async () => {
```

Comment:
As required, When the user clicks on the "Comments" button, the Comments popup appears.

When the comment button is been clicked on, the comment appears on top of the screen instead of a popup, Kindly fix this ✊